python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,13 +41,12 @@ def _getGangaList():
 
 
 class Node(object):
-    _ref_list = ['_parent', '_registry', '_index_cache', '_proxyObject']
+    _ref_list = ['_parent', '_registry', '_index_cache']
```

Comment:
`_proxyObject` is protected from being copied by being in `do_not_copy`. At least it seemed to be based on my tests.